lib/superstore/types.rb

```diff
@@ -1,11 +1,11 @@
-Superstore::Type.register(:array,          Superstore::Types::ArrayType)
-Superstore::Type.register(:boolean,        Superstore::Types::BooleanType)
-Superstore::Type.register(:date,           Superstore::Types::DateType)
-Superstore::Type.register(:date_range,     Superstore::Types::DateRangeType)
-Superstore::Type.register(:float,          Superstore::Types::FloatType)
-Superstore::Type.register(:geo_point,      Superstore::Types::GeoPointType)
-Superstore::Type.register(:integer,        Superstore::Types::IntegerType)
-Superstore::Type.register(:integer_range,  Superstore::Types::IntegerRangeType)
-Superstore::Type.register(:json,           Superstore::Types::JsonType)
-Superstore::Type.register(:time,           Superstore::Types::TimeType)
-Superstore::Type.register(:string,         Superstore::Types::StringType)
+ActiveRecord::Type.register(:superstore_array,          Superstore::Types::ArrayType)
+ActiveRecord::Type.register(:superstore_boolean,        Superstore::Types::BooleanType)
+ActiveRecord::Type.register(:superstore_date,           Superstore::Types::DateType)
+ActiveRecord::Type.register(:superstore_date_range,     Superstore::Types::DateRangeType)
+ActiveRecord::Type.register(:superstore_float,          Superstore::Types::FloatType)
+ActiveRecord::Type.register(:superstore_geo_point,      Superstore::Types::GeoPointType)
+ActiveRecord::Type.register(:superstore_integer,        Superstore::Types::IntegerType)
+ActiveRecord::Type.register(:superstore_integer_range,  Superstore::Types::IntegerRangeType)
+ActiveRecord::Type.register(:superstore_json,           Superstore::Types::JsonType)
+ActiveRecord::Type.register(:superstore_time,           Superstore::Types::TimeType)
+ActiveRecord::Type.register(:superstore_string,         Superstore::Types::StringType)
```

Comment:
Can any of these be deleted? Maybe another time?

Reply:
The range and times are required:

* Storing a range as a json tuple is unique feature
* The date format should not rely on the db format. I do not want all of our json times accidentally change to "2019-02-11 11:33:55".

I'm trying to be careful here. Main goal is to get onto AttributeSet.